{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python-oslo.middleware",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "\nThis update for python-oslo.middleware fixes the following issues:\n\nSecurity issue fixed:\n- CVE-2017-2592: Using the CatchError class may include sensitive values in the error message\n  accompanying a Traceback, resulting in their disclosure (bsc#1022043).\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "SUSE-OpenStack-Cloud-6-2017-477",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2017_0848-1.json"
			},
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2017:0848-1",
				"url": "https://www.suse.com/support/update/announcement/2017/suse-su-20170848-1/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2017:0848-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2017-March/002744.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1022043",
				"url": "https://bugzilla.suse.com/1022043"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2017-2592 page",
				"url": "https://www.suse.com/security/cve/CVE-2017-2592/"
			}
		],
		"title": "Security update for python-oslo.middleware",
		"tracking": {
			"current_release_date": "2017-03-29T11:01:22Z",
			"generator": {
				"date": "2017-03-29T11:01:22Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2017:0848-1",
			"initial_release_date": "2017-03-29T11:01:22Z",
			"revision_history": [
				{
					"date": "2017-03-29T11:01:22Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-oslo.middleware-2.8.0-3.1.noarch",
								"product": {
									"name": "python-oslo.middleware-2.8.0-3.1.noarch",
									"product_id": "python-oslo.middleware-2.8.0-3.1.noarch"
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 6",
								"product": {
									"name": "SUSE OpenStack Cloud 6",
									"product_id": "SUSE OpenStack Cloud 6",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:6"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-oslo.middleware-2.8.0-3.1.noarch as component of SUSE OpenStack Cloud 6",
					"product_id": "SUSE OpenStack Cloud 6:python-oslo.middleware-2.8.0-3.1.noarch"
				},
				"product_reference": "python-oslo.middleware-2.8.0-3.1.noarch",
				"relates_to_product_reference": "SUSE OpenStack Cloud 6"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2017-2592",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2017-2592"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "python-oslo-middleware before versions 3.8.1, 3.19.1, 3.23.1 is vulnerable to an information disclosure. Software using the CatchError class could include sensitive values in a traceback's error message. System users could exploit this flaw to obtain sensitive information from OpenStack component error logs (for example, keystone tokens).",
					"title": "CVE description"
				}
			],
			"product_status": {
				"fixed": [
					"SUSE OpenStack Cloud 6:python-oslo.middleware-2.8.0-3.1.noarch"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2017-2592",
					"url": "https://www.suse.com/security/cve/CVE-2017-2592"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1022043 for CVE-2017-2592",
					"url": "https://bugzilla.suse.com/1022043"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1022152 for CVE-2017-2592",
					"url": "https://bugzilla.suse.com/1022152"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE OpenStack Cloud 6:python-oslo.middleware-2.8.0-3.1.noarch"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.5,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N",
						"version": "3.0"
					},
					"products": [
						"SUSE OpenStack Cloud 6:python-oslo.middleware-2.8.0-3.1.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2017-03-29T11:01:22Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2017-2592"
		}
	]
}
